keyboards/split75/config.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#define VENDOR_ID       0x20A0
```

Comment:
This vender ID appears to already be assigned, to Flirc/Clay Logic, and a different VID should be used

Reply:
Should be good now https://www.the-sz.com/products/usbid/index.php?v=0x594d&p=&n=